templates/helpers/edit-link.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function(branch, path, file) {
+	/* todo - we could check if the file is a symlink and rewrite the path to the target. 
```

Comment:
Declare `var content, base;` at the top of the function otherwise those variables will become global variables (yay, Javascript!)